trimChamber.py

```diff
@@ -86,8 +86,10 @@
             "-s%d"%(options.slot),
             "-g%d"%(options.gtx),
             "--filename=%s"%(filename0),
-	        "--vfatmask=%i"%(options.vfatmask)]
-           )
+            "--vfatmask=%i"%(options.vfatmask),
```

Comment:
Functionality now included in chamber_vfatMask dictionary and used by run_scans.py